applications/api/app/config/controllers/course.yml

```diff
@@ -0,0 +1,5 @@
+services:
+
+  codely.api.controller.course.post:
+    class: CodelyTv\Api\Controller\Course\CoursePostController
+    parent: codely.api.controller
```

Comment:
New line at the end of files by default please 🙏

You can enable it in PhpStorm / IntelliJ:

![preferences 2017-10-13 12-35-54](https://user-images.githubusercontent.com/986235/31542632-14263700-b013-11e7-96e3-bdd96ce7671a.png)

Reply:
Thanks for the tip!